fibonacci.asm

```diff
@@ -16,14 +16,29 @@
 
 	.data
 
-fibs:	.word   0 : 9         # create an array variable named "fibs" of 9 word-length elements (4 bytes each)
-size:	.word  9              # create a single integer variable named "size" that indicates the length of the array
+fibs:	.word  0 : 9        							  # create an array variable named "fibs" of 9 word-length elements (4 bytes each)
+size:	.word  9            							  # create a single integer variable named "size" that indicates the length of the array
+prmpt:	.asciiz "How many Fibonacci numbers would you like to calculate? (1-9): " # message to prompt the user for how many numbers they would like to generate. 
```

Comment:
Your prompt is off by 1 (asks for input in range 1 - 9 but accepts 0), but no pts. lost.